integration/framework/framework.go

```diff
@@ -163,6 +168,28 @@ func (self *realFramework) T() *testing.T {
 	return self.t
 }
 
+func (self *realFramework) GetGoRoutines() (int, error) {
+
+	pprofURL := fmt.Sprintf("%s/debug/pprof", self.Hostname().FullHostname())
+	resp, err := http.Get(pprofURL)
+	if err != nil {
+		return -1, err
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return -1, err
+	}
+	r := regexp.MustCompile("right>([0-9]+)<td")
+	for _, line := range strings.Split(string(body), "\n") {
+		// <tr><td align=right>42<td><a href="goroutine?debug=1">goroutine</a>
```

Comment:
Golang is yet to formalize a format for pprof data. I don't expect golang to change this format anytime soon. 
This is a test. As long as the data is valid, do we really care about how we get the data?